### DISCUSSION TITLE
SCENIC vs SCENIC+

### DISCUSSION THREAD

**Discussion by ghost** [aertslab/scenicplus#177] [Announcements]
Does SCENIC+ do everything SCENIC do? 

Is there anything SCENIC that SCENIC+ does not do?

**Comment by SeppeDeWinter** [comment]
Hi

One of the big differences that SCENIC does but SCENIC+ does not is running motif enrichment analysis using a set of genes. SCENIC+ needs sets of genomic regions.

What are you particularly looking for?

Best,

Seppe

**Comment by ghost** [comment]
Just to make sure, can you expand on what you mean by "motif enrichment analysis" ? 

I think the comparison I am looking for is between SCENIC+ and pySCENIC specifically

I am looking to deduce Transcription factors objectives using scRNA-seq data by examining the correlation between a specific Transcription factors and a gene, as well as the presence of the Transcription factors binding site motif in the gene's promoter region. Next, calculate an AUCell score, representing module activity, for each module in every individual cell. Then employe a ranksum test to compare the AUCell scores of perturbed cells (CRA or CRI) with their corresponding NT cells. Is this possible to do this with SCENIC+. or can it only be done in pySCENIC?

Can we perform similiar steps in the notebooks/pySCENIC - **Full pipeline.ipynb** notebook from the pySCENIC repo?
(https://github.com/aertslab/pySCENIC/blob/master/notebooks/pySCENIC%20-%20Full%20pipeline.ipynb)

**Comment by SeppeDeWinter** [comment]
Motif enrichment analysis: with this we analyse the DNA sequence of a set of regions to test which motifs are overrepresented (enriched) in this set of regions compared to a background.

If you only have scRNA-seq data (and no scATAC-seq data) I would suggest to use pySCENIC instead. SCENIC+ is really designed for combined scRNA-seq and scATAC-seq data.

Best,

Seppe

**Comment by ghost** [comment]
Got it!

So pySCENIC does motif enrichment analysis but SCENIC+ does not.

Are there a default scATAC-seq data files for humans and mouse in SCENIC+?

Are the TF files different for humans and mouse between SCENIC+ and pySCENIC? There are no motif or motif annoation files in SCENIC+? (https://github.com/aertslab/pySCENIC/tree/master/notebooks/data)

If we only use scRNA-seq data and do not provide SCENIC+ with scATAC-seq data will a "Default" scATAC-seq data be used for mouse and humans?

**Comment by SeppeDeWinter** [comment]
Both do motif enrichment analysis.

SCENIC does on regions surrounding each gene (using a standard distance). SCENIC+ does it on ATAC-seq peaks.

There are no default ATAC-seq files. They are always dataset specific (the ATAC peaks vary a lot across systems).

The TF files are the same for both tools also the motifs and motif annotation files used are the same for both tools.

I hope this clears things up. Best Seppe

**Reply by ghost** [reply]
it does thank you!

**Comment by weixiwency** [comment]
further more is there anything SCENIC+ do but SCENIC does not do?  besides the difference on the ATAC-seq
